src/Symfony/Component/Workflow/Definition.php

```diff
@@ -17,8 +17,9 @@
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
  */
-class Definition
+final class Definition
```

Comment:
Please next time don't ever do this without providing abstraction that final class implements.